ReleaseNotes.md

```diff
@@ -1,5 +1,55 @@
 # Release Notes
 
+## New in Release 2.5.0
+Post-training Quantization:
+- Features:
+  - Added `"overflow_fix"` parameter support (improved accuracy for optimized model for affected devices).
+  - Added `"model_type"` parameter support that allows reach better accuracy for specific model architectures (e.g. transformers).
+  - Added `"inplace_statistics"` support (reduce memory footprint during optimization).
+  - Added `nncf.parameters.IgnoredScope` parameter support that allows exclude layers from optimization scope in different ways.
+  - (OpenVINO) Introduced post-training support for backend.
```

Comment:
I would put it as the first bullet after "Features" and rephrase it to 
- Added Post-training Quantization support for OpenVINO IR (OpenVINO backend).

Reply:
Done